Terminal.Gui/View/ViewDrawing.cs

```diff
@@ -454,7 +454,7 @@ public virtual void OnDrawContent (Rect contentArea)
 	{
 		if (NeedsDisplay) {
 			if (SuperView != null) {
-				Clear (BoundsToScreen (Bounds));
+				Clear (BoundsToScreen (contentArea));
```

Comment:
`contentArea` is the same as `Bounds` but only screen relative, right?

Reply:
Yes. I was going to remove that param from OnDrawContent but now I'm glad i never did. 

You'll see why soon...